beacon-chain/core/state/state_transition.go

```diff
@@ -123,6 +123,15 @@ func NewStateTransition(
 	return newState, nil
 }
 
+func NewSlotTransition(state *types.BeaconState, block *types.Block) *types.BeaconState {
+	_ = block
```

Comment:
What's going on with these two methods?

Reply:
These are placeholders to implement in the next pr, for slot transition and epoch transition.

Reply:
Please add comments/TODOs, this just looks like unfinished code

Reply:
Sure thing, will add it